CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 
 ### Fixed
 - C#: Parse attributes for local functions (#3348)
+- Go: Recognize other common package naming conventions (#2424)
```

Comment:
an example could be nice. Those changelog entries can be hard to understand and I guess some people just read that to check what is new.